discovery/seata-discovery-raft/src/main/java/org/apache/seata/discovery/registry/raft/RaftRegistryServiceImpl.java

```diff
@@ -377,14 +377,14 @@ private static void acquireClusterMetaData(String clusterName, String group) thr
             try (CloseableHttpResponse httpResponse =
                 HttpClientUtil.doGet("http://" + tcAddress + "/metadata/v1/cluster", param, header, 1000)) {
                 if (httpResponse != null) {
-                    if (httpResponse.getStatusLine().getStatusCode() == HttpStatus.SC_OK) {
+                    int statusCode = httpResponse.getStatusLine().getStatusCode();
+                    if (statusCode == HttpStatus.SC_OK) {
                         response = EntityUtils.toString(httpResponse.getEntity(), StandardCharsets.UTF_8);
-                    } else if (httpResponse.getStatusLine().getStatusCode() == HttpStatus.SC_UNAUTHORIZED) {
-                        if (StringUtils.isNotBlank(USERNAME) && StringUtils.isNotBlank(PASSWORD)) {
-                            throw new RetryableException("Authentication failed!");
-                        } else {
-                            throw new AuthenticationFailedException("Authentication failed! you should configure the correct username and password.");
-                        }
+                    } else if (statusCode == HttpStatus.SC_UNAUTHORIZED) {
+                        refreshToken(tcAddress);
```

Comment:
why remove 
`(StringUtils.isNotBlank(USERNAME) && StringUtils.isNotBlank(PASSWORD)`

Reply:
I thought it wasn't necessary, but having validation might be good, so I made the changes.

Reply:
> I thought it wasn't necessary, but having validation might be good, so I made the changes.

I think refreshToken(tcAddress); should be placed after the condition check; otherwise, it will waste a call when the account password is empty

Reply:
I initially placed it outside the condition because refreshToken checks the account, but your point makes sense, so I moved it. Thanks for the feedback!